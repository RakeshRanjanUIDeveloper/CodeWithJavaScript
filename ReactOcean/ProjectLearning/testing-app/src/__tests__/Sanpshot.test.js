import { render } from "@testing-library/react"
import Snapshot from "../components/Snapshot"

test("snapshot for snapshot component", ()=>{
    const { asFragment } = render(<Snapshot />);
    expect(asFragment()).toMatchSnapshot();

})