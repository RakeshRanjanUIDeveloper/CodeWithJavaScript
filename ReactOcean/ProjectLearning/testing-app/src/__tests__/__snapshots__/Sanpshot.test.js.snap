// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot for snapshot component 1`] = `
<DocumentFragment>
  <div>
    Snapshot
  </div>
</DocumentFragment>
`;
